### PR TITLE
[FontImporter] Add support for importing more characters from files.

### PR DESCRIPTION
**Pull request by lihaochen910**
fonts.muder
```
# Hi there! If you want to load .ttf fonts use the following format: 
# <index> <path-to-font>: <font-size> [<offsetX> <offsetY>] [charsDefineFile]
# For example: 
#   100 FontName.ttf: 16
#   101 FontName.ttf: 16 0 0 chars_define.txt
```
image

### CODE REVIEW

File: src/Murder.Editor/Data/Graphics/RectpackSharp.cs [hunk 1]
```diff
@@ -0,0 +1,769 @@
...
+        /// Sorts the given <see cref="PackingRectangle"/> array using the specified <see cref="PackingHint"/>.
+        /// </summary>
+        /// <param name="rectangles">The rectangles to sort.</param>
+        /// <param name="packingHint">The hint to sort by. Must be a single bit value.</param>
+        /// <remarks>
+        /// The <see cref="PackingRectangle.SortKey"/> values will be modified.
+        /// </remarks>
+#if NET5_0_OR_GREATER
+        public static void SortByPackingHint(Span<PackingRectangle> rectangles, PackingHints packingHint)
+#elif NETSTANDARD2_0
```
Comment (isadorasophia) [inline src/Murder.Editor/Data/Graphics/RectpackSharp.cs:99]: I think we can remove these `#ifdefs`? Since Murder only targets .NET 8.

Comment (lihaochen910) [reply src/Murder.Editor/Data/Graphics/RectpackSharp.cs:99]: [RectpackSharp](https://github.com/ThomasMiz/RectpackSharp) is referenced from another library's implementation, and I'm wondering if I should reference its nuget package or would it be better to put it in a separate file?

Comment (lihaochen910) [reply src/Murder.Editor/Data/Graphics/RectpackSharp.cs:99]: If you don't want to add nuget package references, I think you can remove those "#ifdefs"

**Comment by lihaochen910** [comment]
@isadorasophia, I removed those # ifdefs in this [submission](https://github.com/isadorasophia/murder/pull/70/commits/22dface8882c0003ae34499759b1ef68e0cf4d78) and now only support dotnet 5 or above

**Review by isadorasophia — APPROVED** [review]
Thank you for the fix! :) There's some cleanups that we can do around the `goto` (it makes things slightly less readable), but we can fix that after it's checked-in on our side!

**Pull request merged**